packages/orbit-components/src/tmp-tailwind/Switch/index.tsx

```diff
@@ -0,0 +1,59 @@
+"use client";
+
+import * as React from "react";
+import cx from "clsx";
+
+import Circle from "../../icons/Circle";
+import handleKeyDown from "../../utils/handleKeyDown";
+import type { Props } from "../../Switch/types";
+
+const Switch = React.forwardRef<HTMLInputElement, Props>(
+  ({ onChange, checked, dataTest, id, icon, onBlur, onFocus, disabled, ariaLabelledby }, ref) => {
+    return (
+      <div className="inline-block">
+        <div
+          className={cx(
+            "duration-fast relative flex h-[20px] w-[40px] items-center justify-between rounded-[100px] transition-colors",
+            disabled ? "opacity-50" : "cursor-pointer",
+            checked ? "bg-blue-normal" : "bg-cloud-dark",
+          )}
+        >
+          <input
+            className="peer absolute inset-0 h-full w-full opacity-0"
```

Comment:
Here you omitted `margin` and `padding` set to 0. That's fine, as it's 0 in the computed styles due to the preflight styles (I might be mistaken on this, not 100% sure).

![image](https://github.com/kiwicom/orbit/assets/8863238/5c781772-aa43-4cf8-b781-8c5c4ef173b6)

What if consumers disable the preflight setting? Won't this affect them? As input checkbox has margin and padding by default. [Playground](https://codi.link//PHA+CjxpbnB1dCB0eXBlPSJjaGVja2JveCIgcm9sZT0ic3dpdGNoIiAvPmZvbwo8L3A+CjxwPgo8aW5wdXQgY2xhc3M9ImlucHV0IiB0eXBlPSJjaGVja2JveCIgcm9sZT0ic3dpdGNoIiAvPmJhcgo8L3A+Cg==|LmlucHV0IHsKICBtYXJnaW46IDA7CiAgcGFkZGluZzogMDsKfSA=|).

Reply:
Honestly, I don't remember thinking about that when (not) adding the margin and padding classes.
I also don't know if they are not somehow useful for when using a Switch inside other layout components…
I prefer to have them rather than relying on preflight. Adding them back. Well spotted.